frontend/src/layout/navigation-3000/Navigation.scss

```diff
@@ -43,24 +43,18 @@
     // `relative` is for positioning of the scene-level spinner
     position: relative;
     min-height: calc(100vh - var(--breadcrumbs-height-full) - var(--scene-padding) - var(--scene-padding-bottom));
+    padding-bottom: 20rem; // Breathing room for our products so they're not hugging the bottom of the screen
```

Comment:
We already have a setting for this, right above: the `--scene-padding-bottom` var. It's the same as `var(--scene-padding)` by default, but we can increase that.

The massive 20rem value looks a bit sloppy to my eye, it's extremely loose. You're right that the current value is super tight though. I propose we set `--scene-padding-bottom: 4rem` – some more breathing room, but in harmony with the padding on other sides.